tests/test_dataset.py

```diff
@@ -271,16 +271,17 @@ def test_dataset_append_local(CLIENT, dataset):
         DatasetItem(
             image_location=LOCAL_FILENAME,
             metadata={"test": 0},
-            reference_id=LOCAL_FILENAME.split("/")[-1],
+            reference_id=LOCAL_FILENAME.split("/")[-1] + str(i),
         )
+        for i in range(1000)
     ]
 
     response = dataset.append(ds_items_local)
 
     assert isinstance(response, UploadResponse)
     resp_json = response.json()
     assert resp_json[DATASET_ID_KEY] == dataset.id
-    assert resp_json[NEW_ITEMS] == 1
+    assert resp_json[NEW_ITEMS] == 1000
```

Comment:
This should be refactored to a common constant with the constructor.

Reply:
not sure which constructor you  mean but I did refactor out a constant